docs/source/refs.bib

```diff
@@ -1,214 +1,183 @@
 
 @article{Wang:2017go,
```

Comment:
What tool did you use for reformatting this btw?

Reply:
I didn't use a specific tool to reformat, just whatever the built in linter for VSCode is. I also manually deleted a lot of the categories that we would never want to display (e.g. abstracts) and the stuff that doesn't relate to the reference itself (e.g. paths to local storage). Is there a more sensible way I should have done this?

Reply:
Not really. I just wondered what you'd change and whether you had a useful tool I hadn't heard of.

I've found JabRef is really handy for working with bibtex files: jabref.org

It does have some built-in tools for fixing broken tags etc.